src/main/java/com/thevoxelbox/voxelsniper/brush/type/RegenerateChunkBrush.java

```diff
@@ -25,8 +25,12 @@ private void generateChunk(Snipe snipe) {
         int chunkX = targetBlock.getX() >> 4;
         int chunkZ = targetBlock.getZ() >> 4;
         SnipeMessenger messenger = snipe.createMessenger();
-        messenger.sendMessage("Generate that chunk! " + chunkX + " " + chunkZ);
-        regenerateChunk(chunkX, chunkZ);
+        messenger.sendMessage("Generating that chunk! " + chunkX + " " + chunkZ);
```

Comment:
Maybe leave a note that it could take some, depending on the world type (flat/augmented) this could take a bit longer.